.github/workflows/ci.yml

```diff
@@ -36,6 +36,18 @@ jobs:
             java: adoptium@11
           - ci: ciNodeJS
             java: adoptium@17
+          - ci: ciJVM
+            scala: 2.12.15
+            java: adoptium@11
+          - ci: ciJVM
+            scala: 2.12.15
+            java: adoptium@17
+          - ci: ciJVM
+            scala: 3.1.0
+            java: adoptium@11
+          - ci: ciJVM
+            scala: 3.1.0
+            java: adoptium@17
```

Comment:
Should these be re-thinned?

Reply:
Yes.  I didn't notice they were thickened.

Reply:
Wait, is it wrong?  It's running the JVM on each Java for 2.13, on each Scala for Adoptium 8.  And it's running the Node build once each Scala.  And then duplication from pull_request / push.

Reply:
Not sure, but shouldn’t it be the same in 0.23 and main? (cc @armanbilge )

Reply:
Sorry I looked too quickly when I 👍 yesterday, retracted 😉 

This looks correct. It's just confusing (at least for me), b/c the list @bplommer commented on is the _excluded_ job list. If you look below at what CI actually did it's right.

Reply:
So this one is good then?  There's a chonky one coming up behind it.